CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ Changelog
 
 Unreleased
 ----------
+### Added
+* Added test helper to hide a blinking input caret in an element
+
+### Changed
+* The WDIO TerraService automatically hides carets in the page whenever a page is loaded or refreshed
```

Comment:
This should be listed under added since this is a new feature

Reply:
Changed in cce5091cbdebce172e5f5c54d0950f86134a8b33